README.md

```diff
@@ -1,33 +1,53 @@
 [![Build Status](https://travis-ci.org/webrtc/testrtc.svg?branch=master)](https://travis-ci.org/webrtc/testrtc)
-# TestRTC #
-
-Collection of tests and tools to help diagnose WebRTC systems.
-
-[WebRTC troubleshooter](https://test.webrtc.org/)
 
+# TestRTC #
+[WebRTC troubleshooter](https://test.webrtc.org/) provides a set of tests that can be easily run by a user to help diagnose
+WebRTC related issues. The user can then download a report containing all the gathered information or upload the log and
+create a temporary link with the report result.
+
+## Automatic tests ##
+* Microphone
+  * Audio capture
+    * Checks the microphone is able to produce 2 seconds of non-silent audio
+    * Computes peak level and maximum RMS
+    * Clip detection
+    * Mono mic detection
+* Camera
+  * Check WxH resolution
+    * Checks the camera is able to capture at the requested resolution for 5 seconds
+    * Checks if the frames are frozen or muted/black
+    * Detects how long to start encode frames
+    * Reports encode time and average framerate
+  * Check supported resolutions
+    * Lists other resolutions that appear to be supported
```

Comment:
It actually checks the same resolutions as "Check WxH resolution" does as well. Not sure if that should be changed in the actual test or in this text.